AutomatorServer/ios/AutomatorServerUITests/PatrolAutomation.swift

```diff
@@ -160,24 +160,33 @@ class PatrolAutomation {
   func allowPermissionWhileUsingApp() {
     runAction("allowing while using app") {
       let systemAlerts = self.springboard.alerts
-      let okButton = systemAlerts.buttons["OK"]
-      if okButton.exists {
-        okButton.tap()
-      }
+      let labels = ["OK", "Allow", "Allow While Using App"]
```

Comment:
Shouldn't `"Allow While Using App"` be first?

Reply:
It doesn't matter.